### PR TITLE
Add query contains only schema validation

### PR DESCRIPTION
**Pull request by fowczarek**
I want to merge this change because adding query contains only schema validation

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2159 Lock option to get introspection and any other data in one request](https://app.clickup.com/t/2549495/SALEOR-2159)

File: saleor/graphql/core/tests/test_view.py [hunk 1]
```diff
@@ -309,3 +309,24 @@ def test_example_query(api_client, product):
     response = api_client.post_graphql(EXAMPLE_QUERY)
     content = get_graphql_content(response)
     assert content["data"]["products"]["edges"][0]["node"]["name"] == product.name
+
+
+def test_query_contains_not_only_schema_raise_error(api_client, graphql_log_handler):
+    response = api_client.post_graphql(
+        """
+        query IntrospectionQuery {
+            me {
```
Comment (korycins) [inline saleor/graphql/core/tests/test_view.py:318]: It is a detail, but I would add parametrize here, to cover also the standard case like `products(first:5)`.
Using `me` here  can suggest that it raises only for protected resources

File: saleor/graphql/views.py [hunk 2]
```diff
@@ -212,6 +212,16 @@ def parse_query(
         except (ValueError, GraphQLSyntaxError) as e:
             return None, ExecutionResult(errors=[e], invalid=True)
 
+    def check_if_query_contains_only_schema(self, document: GraphQLDocument):
+        for definition in document.document_ast.definitions:
+            selections = definition.selection_set.selections
+            if len(selections) > 1:
```
Comment (korycins) [inline saleor/graphql/views.py:218]: If you revert the condition, you will reduce the indentations but it is up to you

**Review by korycins — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Comment by maarcingebala** [comment]
@fowczarek We should add a changelog entry for this.

**Review by IKarbowiak — APPROVED** [review]

**Pull request merged**